### ISSUE TITLE
Top positionning bug

### ISSUE THREAD

**Issue by limax84** [dbushell/Pikaday#310]
When setting the option "position: 'top left'" or "position: 'top right'" the vertical position does not change.

The reason is this line :
L.905:  top - height - field.offsetHeight > 0

The if statement check if the height of the height of the field AND the height of the calendar can be substracted from the "top" attribute which is wrong. ONLY the field height should be substracted.

Correction:
L. 905: top - field.offsetHeight > 0
L. 908: top = top - field.offsetHeight;

**Comment by nathancahill** [comment]
No, I think the math is correct. See here: http://jsfiddle.net/8usv85fk/

Try changing the `margin-top` value to 1px less than the height of the calendar (in my browser 210px). The picker does not display on top. Now set the margin to the height of the calendar. The picker displays on top.

**Comment by san4ezz87** [comment]
I have the same problem. What should I do if I have 3 inputs and I can't do margin 210px between them.
I just want to pikaday to overlay my inputs. Can you help me?
